packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -416,6 +416,68 @@ describe('expression', () => {
         expect(result.warnings).toHaveLength(0);
     });
 
+    it('forbids extra curly brackets on text nodes', () => {
+        // Note: the special case with curly brackets on text nodes is because of Text node traversal does not properly
+        // builds the expression: const tokenizedContent = rawText.split(EXPRESSION_RE);
+        // converting {{foo}} into {{foo}, and the later is what's passed into the expression parser.
+        let result = parseTemplate(`<template>{{foo}}</template>`);
```

Comment:
Instead of adding more tests to `parser.spec.ts`, can we move those tests to a fixture?
I would like to remove this file in the long run.